src/renderer/pages/preferences-page.js

```diff
@@ -205,6 +208,22 @@ class PreferencesPage extends React.Component {
     )
   }
 
+  soundNotificationsCheckbox () {
+    return (
+      <Preference>
+        <Checkbox
+          className='control'
+          checked={this.props.state.saved.prefs.soundNotifications}
+          label={'Activate the notification sounds'}
```

Comment:
Nitpick: I would prefer 'Enable notification sounds'.

But since it is both interaction sounds and notification sounds, maybe it should simply be 'Enable sounds'?